auth.php

```diff
@@ -0,0 +1,26 @@
+<?php
+require_once('php/connection.php');
+function validateUser($email, $password) {
+    $sql = "CALL GetUserFromEmail(:email);";
+    $conn = newConn();
+    $stmt = $conn->prepare($sql);
+    $stmt->bindValue(":email", $email, PDO::PARAM_STR);
+    $stmt->execute();
+    $data = $stmt->fetch();
+    $conn = null;
+    if ($data === null || $data === false) return false;
+    if (!password_verify($password, $data['password'])) return false;
+   return$data;
```

Comment:
Add a check to see if the user's 'state' is 'active', and if it isn't then don't allow them to log in.

Reply:
Can you please state exactly what I need to do.

Reply:
Because I struggling to get that done.

Reply:
`if ($data['state'] !== 'active') return false;`